src/txs/Tx.tsx

```diff
@@ -313,10 +315,10 @@ function Tx<TxValues>(props: Props<TxValues>) {
     return (
       <button
         type="button"
-        className={classNames({ muted: !isMax })}
+        className={cx({ muted: !isMax })}
         onClick={onClick ? () => onClick(max) : () => setIsMax(!isMax)}
       >
-        <Flex gap={4} start>
+        <Flex gap={4}>
```

Comment:
Something is causing a `Something went wrong` error on the `Send` page.

Reply:
whats the error? not getting this on my end

Reply:
This issue doesn't seem to do with this branch.  Created a ticket.